File: Myserver/Maserver/src/com/ma/server/HandlerMap.java
package com.ma.server;

import java.util.HashMap;
import java.util.Map;
import com.ma.handler.MyHandler;

public class HandlerMap {
 
	private static Map<Short, MyHandler> Handlers=new HashMap<Short, MyHandler>(); ;  
	  
	public static MyHandler getHandlersByType(short type) {  
	    return  Handlers.get(type);  
	}  
	
	
  public static void registeHandler(short code,MyHandler handler )
  {
	  Handlers.put(code, handler);
	  
  }
}

File: elsfk/Assets/Scripts/Handler/HandlerMap.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class HandlerMap 
{
    private static Dictionary<short, MyHandler> Handlers = new Dictionary<short, MyHandler>(); 
	  
	public static MyHandler getHandlersByType(short type)
    {
        return Handlers[type];
    }


    public static void registeHandler(short code, MyHandler handler)
    {
        Handlers.Add(code, handler);
    }
}

